### ISSUE TITLE
Subreddit menu doesn't appear on certain subreddit, when subreddit manager is disabled

### ISSUE THREAD

**Issue by fireattack** [honestbleeps/Reddit-Enhancement-Suite#3382]
Can reproduce on two accounts.

Sub in question: https://www.reddit.com/r/DotA2/

Screenshot:

![image](https://cloud.githubusercontent.com/assets/4130991/18228840/77461cd8-7225-11e6-8b40-1bc4849c16d2.png)

**Comment by fireattack** [comment]
I did some further investigation. I disabled *all* the modules I can disable, but the problem persists.

Could anyone at least confirm this bug?

**Comment by andytuba** [comment]
If you've disabled RES's Subreddit Manager and issue occurs in certain subreddits, then it's a problem with the subreddit stylesheet and you should send modmail asking for a fix. Before modmailing, it's worth disabling RES entirely first and double-checking the problem still occurs.

**Comment by fireattack** [comment]
@andytuba 

Like I said before, I think it's caused by RES because it can *not* be reproduced when RES is entirely disabled and can be produced with only RES extension installed.

**Comment by andytuba** [comment]
Thanks for clarifying that it goes away when RES is disabled entirely. Sounds like there should be some subreddit manager CSS which is wrapped inside a subreddit manager bodyclass.

**Comment by mrkorb** [comment]
I believe this is caused by the pinHeader option (Appearance > betteReddit) being set to either the 'Subreddit Bar only' or 'Subreddit Bar and User bar' option.  If you toggle it to any of the other options, the menu opens up just fine, but if you select either of those 'Subreddit bar' options, it does not open, regardless of the custom CSS (or lack of) on a subreddit.

**Comment by fireattack** [comment]
No, it's not relavant. Mine is "none", and changing it to any other option doesn't affect/fix this problem.

**Comment by fireattack** [comment]
It happens on this subreddit too: https://www.reddit.com/r/instant_regret/comments/5cbksh/damn_it/
